web-model/src/main/java/com/oasis/webmodel/response/LoginResponseModelOK.java

```diff
@@ -0,0 +1,12 @@
+package com.oasis.webmodel.response;
+
+import lombok.Data;
+
+@Data
+public class LoginResponseModelOK {
```

Comment:
no all capital case,
change it to `LoginSuccessResponse`

Reply:
Commit ID: 279a1f3